compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -435,8 +434,8 @@ public STNode resumeParsing(ParserRuleContext context, Object... args) {
                 return parseWaitFieldEnd();
             case ANNOT_CHAINING_TOKEN:
                 return parseAnnotChainingToken();
-            case ANNOT_TAG_REFERENCE:
-                return parseAnnotTagReference();
+            case QUALIFIED_IDENTIFIER_OR_IDENTIFIER:
+                return parseQualifiedIdentifierOrIdentifier();
```

Comment:
Would something like "field-access-identifier" be a better word for both context and method name? because qualified-identifier is a bit generic..